.github/workflows/old_os.yml

```diff
@@ -1,189 +0,0 @@
-name: Build and test for Ubuntu 18.04
```

Comment:
Could you explain why this file is being removed?

Reply:
Because GitHub action no longer supports ubuntu 18.04. This action can not be executed now.

Reply:
Ah okay

Reply:
There seems to be a workaround for this problem. Ref: https://github.com/LuaLS/lua-language-server/pull/1835/files